srcpkgs/deadbeef/template

```diff
@@ -1,31 +1,24 @@
 # Template file for 'deadbeef'
 pkgname=deadbeef
-version=1.8.8
+version=1.9.5
 revision=1
 create_wrksrc=yes
 build_style=gnu-configure
-configure_args="--disable-oss --disable-lfm --disable-notify --disable-gtk2"
+configure_args="--disable-oss --disable-notify --disable-gtk2"
 hostmakedepends="automake libtool gettext gettext-devel intltool pkg-config
- yasm clang glib-devel"
+ yasm clang glib-devel libdispatch"
```

Comment:
I don't see anything in that package that would be used on the host.

Reply:
It must be libdispatch-devel